### DISCUSSION TITLE
Choose a strategy for Print objects with volumes (single/multiple entitles)

### DISCUSSION THREAD

**Discussion by mark-rozen** [PanditProject/Website#16] [Help]
Hello PANDiT community,

I've noticed the following guide note during the editing of one of the work entities.
<img width="1187" alt="Screen Shot 2021-05-21 at 16 42 59" src="https://user-images.githubusercontent.com/55999348/119155628-ac422500-ba53-11eb-8f8e-cf76ece0a28f.png">

From that, I understood that if a book is published in several volumes, so it means that each volume has to be a separate print entity.
I'd like to know if it is obligatory (like there is a previous decision to do so)? I’m asking because sometimes it’s impossible without having an actual copy of the volumes to know when each of them was published. For example, in WorldCat, often only a range of the years between the first and the last volumes is provided. Also, I wonder, does it really make sense to create a separate entity for each volume of a book, when all of them use the same title. I can understand the usefulness of it when volumes have different titles, but otherwise...

[@AmirSimantov removed text from here to a new discussion: #17]

I would like to hear your comments on this topic.
Thank you!

**Comment by AmirSimantov** [comment]
Thank you very much for going public with this discussion, @mark-rozen, continuing our email correspondence.\
@chakrabortydeepro, you were also in the loop of the emails.

I am copying (with some changes) the text from an email between @ybronner and me to which we haven't got yet:

Currently, each volume of a multi-part print object should have its own Print entity (see the configuration spreadsheet [Print kinds tab](https://docs.google.com/spreadsheets/d/1SLjJyuPEU8BXfschex56HGRAA9M3AKjMWWlyq7cJf88/edit#gid=94)).

However, we don't have a relationship between a Print entity and a parent/including Print entity. I am not sure that we should, though. If we do, we should consider if a contributor can refer to a parent Print entity from the "Scholarship" and "Editions" fields or only to a child one (and may prevent more than one level of hierarchy). We should also make it clear in those referring fields that the "Location in print"/"Note" inner field should include only the page(s), not the volume.

Here are some examples where I can see "vol" in the scholarship's notes:
[entity/109637/manuscript](https://www.panditproject.org/entity/109637/manuscript)
[entity/109088/manuscript](https://www.panditproject.org/entity/109088/manuscript)
[entity/109062/manuscript](https://www.panditproject.org/entity/109062/manuscript)
[entity/108998/manuscript](https://www.panditproject.org/entity/108998/manuscript)

--

We also have stumbled upon where we have imported data from SKSEC, which we should consider.

**Comment by wujastyk** [comment]
I definitely agree with Mark's point about having one entry for
multi-volume works.  I've already been doing that for NCC, for example (39
volumes and still growing!) and Meulenbeld's HIML.  Other candidates are
the RORI and Madras Descriptive catalogues, again, scores of volumes each.
In this world of manuscripts and bibliographies, vast multi-volume
publications are common.  (VOHD 2, Thanjavur, Maisur, Trivandrum, Varanasi,
BORI, somebody stop me!)  These must all be referred to as single entities,
with volume and page numbers as necessary.  To treat these hundreds of
books as individual items would lead to much confusion, poor quality
data-entry, and much subsequent manual data-laundering.

The catalogue of catalogues by Biswas et al.
<https://wujastyk.net/mscats/Sanskrit_Catalogues/Biswas/> is a superb piece
of bibliographical skill and begs to be leveraged in a system like
PanditProject.

Best,
Dominik

On Fri, 21 May 2021 at 08:57, Mark Rozenkrants ***@***.***>
wrote:

> Hello PANDiT community,
> [...]

From that, I understood that if a book is published in several volumes, so
> it means that for each volume has to be a separate print entity.
> I'd like to know if it is obligatory (like there is a previous decision to
> do so)? I’m asking because sometimes it’s impossible without having an
> actual copy of the volumes to know when each of them was published. For
> example, in WorldCat, often only a range of the years between the first and
> the last volumes is provided. Also, I wonder, does it really make sense to
> create a separate entity for each volume of a book, when all of them use
> the same title. I can understand the usefulness of it when volumes have
> different titles, but otherwise...
>

**Comment by AmirSimantov** [comment]
I think the above suggestion is good: to have only one Print entity for a multiple-volume source. All the reasons make sense to me.

However, there are some existing data that we should pay attention to.

If we go with this change, we should remove the "Volume" field from the Print entity type. However, there are already 400+ Print entities with the "Volume" field filled. I think that the use of them is ambiguous:

1. The value is sometimes the volume number (examples: "4", "XXXI").
2. In other times, The value states some volumes (examples: "7 vols", "2 vols (9 parts)").

At least for the first use, before removing that field, each entity that **refers** to any of them should be revised; in many of them, we should add the volume number to the "Scholarship -> Location in print" inner field.

I have prepared two views that can help us revise the existing data;

350+ entities refer to such Print entities using the "Scholarship" composite field; see:
[Entities referring by **"Scholarship"** to Print entities with the "Volume" field filled](https://www.panditproject.org/admin/entities-referring-by-scholarship-to-print-entities-with-the-volume-field-filled)

About 800 Manuscript entities using the "Catalogs" composite field; see:
[Entities referring by **"Catalogs"** to Print entities with the "Volume" field filled](https://www.panditproject.org/admin/entities-referring-by-catalog-to-print-entities-with-the-volume-field-filled?order=field_volume&sort=asc)

We should also define conventions of how to use the "Scholarship/Catalogs -> Location in print" inner field. Other terms I have found in them are "Section," "Part"; we should take them into account (sure, according to already well-established bibliography references).

I hope that I haven't missed something :)

**Reply by mark-rozen** [reply]
I am glad that we found a solution to this issue. However, I have to say something regarding the first way of using the "volume" field you mentioned. Sometimes print entities of kinds such as "In collection, In book, Inproceedings" and etc. use this field to indicate in which volume the material is precisely located if the source has multiple volumes. So, I think there is no need to remove this field. For example, consider this [Potter 1977c](https://www.panditproject.org/entity/62155/print) print entity of "In Collection" print kind.
I agree with you that the second way is not a proper way to use this "volume" field, and this information if necessary could be placed in the field of "Facts".

**Reply by AmirSimantov** [reply]
You are correct; we will leave the "Volume" field for such cases. We may rename it, though.
